### PR TITLE
EFR32MG21 router

### PR DESCRIPTION
**Pull request by BWFCGUIXT**
Added [EFR32MG21 router]

### CODE REVIEW

File: devices/custom_devices_diy.js [hunk 1]
```diff
@@ -816,4 +816,13 @@ module.exports = [
...
                 .withValueMin(0).withValueMax(99)],
     },
+    {
+        zigbeeModel: ['UT-02'],
+        model: 'EFR32MG21.Router',
+        vendor: 'Custom devices (DiY)',
+        description: 'EFR32MG21 Router',
+        fromZigbee: [],
+        toZigbee: [],
+        exposes: [],
```
Comment (Koenkk) [inline devices/custom_devices_diy.js:826]: For the ti.router we setup reporting of the zclVersion attribute such that we occasionally get a linkquality value. Can something be similar done for this device? https://github.com/Koenkk/zigbee-herdsman-converters/blob/ce9d893320e6acf28b6217c6d19bbb1863072b72/devices/custom_devices_diy.js#L273

Comment (BWFCGUIXT) [reply devices/custom_devices_diy.js:826]: Hi Koen,

Sadly I don't really have any experience I just followed your website.
I tried adding the code but I'm getting the following error:

`Failed to configure 'EFR32MG21 Router (Living Room)', attempt 2 (TypeError: Cannot read properties of undefined (reading 'bind') at Object.bind (/app/node_modules/zigbee-herdsman-converters/lib/reporting.js:40:24) at Object.configure (/app/dist/util/externally-loaded.js:20:29) at Configure.configure (/app/lib/extension/configure.ts:115:37) at EventEmitter.<anonymous> (/app/lib/extension/configure.ts:80:62) at EventEmitter.emit (node:events:525:35) at EventBus.emitLastSeenChanged (/app/lib/eventBus.ts:52:22) at Controller.<anonymous> (/app/lib/zigbee.ts:66:27) at Controller.emit (node:events:513:28) at Controller.selfAndDeviceEmit (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:515:14) at Controller.onZclOrRawData (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:730:18))`

Comment (BWFCGUIXT) [reply devices/custom_devices_diy.js:826]: `Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'ConfigureReporting 0x040d84fffe4d8fa3/1 genBasic([{"attribute":"zclVersion","minimumReportInterval":60,"maximumReportInterval":3600,"reportableChange":0}], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 60618 - 1 - 55 - 0 - 7 after 10000ms)'`

Comment (Koenkk) [reply devices/custom_devices_diy.js:826]: Seems the firmware of the device doesn't support it. Maybe you have to enable it via a compilation flag (I remember I had to do the same for the Texas Instruments firwmare)

Comment (Koenkk) [reply devices/custom_devices_diy.js:826]: I see, then we leave it as is. thanks!

**Pull request merged**